### ISSUE TITLE
Bug: Missing Social Media Logos

### ISSUE THREAD

**Issue by shreyyq** [subhadipbhowmik/30-Days-Of-CPP#140]
The website's footer is missing social media logos, which are essential for enhancing user engagement and accessibility to the social media profiles. Adding these icons will improve the overall user experience and connectivity.
Please assign this issue to me under gssoc.
